src/controllers/enterpriseController.ts

```
@@ -104,19 +105,21 @@ export async function postBulkTransfer(
   next: NextFunction,
 ): Promise<void> {
   try {
-    res.status(200).json({
-      totalBalance: null,
-      byCurrency: [],
-      message: "Treasury view not yet implemented.",
-    });
+    const organizationId = req.apiKey?.organizationId ?? undefined;
+    const tolerance = req.query.tolerance
+      ? parseFloat(req.query.tolerance as string)
+      : undefined;
```

ESLint

[error] Replace `⏎······?·parseFloat(req.query.tolerance·as·string)⏎·····` with `·?·parseFloat(req.query.tolerance·as·string)`

(prettier/prettier)